### PR TITLE
Features/86cx9cjzy/layout design user

### PR DESCRIPTION
**Pull request by adi061001**
layout design user

### CODE REVIEW

File: client/src/App.css
Comment (tw-vineet) [inline client/src/App.css:1]: Remove unwanted code in file

File: client/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+
+
+
+
+export default function App() {
+  return (<>
+    
+      {/* <Navbar/> */}
```
Comment (tw-vineet) [inline client/src/App.tsx:16]: Please remove comment code

File: client/src/Dto/dataType.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    loginPassword:string
+}
+export interface signUpDetails{
+    userName:String
+    userEmail:String
+    userPhone:String
+    userPassword:String
+}
+
+export interface buttonCs {
```
Comment (tw-vineet) [inline client/src/Dto/dataType.ts:12]: Change button naming convention

File: client/src/Layout/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  background-color: black;
+  padding: 30px 0px 30px 0px;
+
+`
+const FooterInput = styled(Box)`
+  border: 1px solid white;
+  width: 210px;
+  height: 46px;
+  margin-top: 16px;
+`
```
Comment (tw-vineet) [inline client/src/Layout/Footer.tsx:22]: Please replace to `const FooterInput = styled()(({ theme }) => ({

}))`

File: client/public/logo512.png
Comment (shubh123thoughtwin) [inline client/public/logo512.png:1]: Remove unused css Images.

File: client/src/Layout/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+export default function Footer() {
+  return (
+    <>
+      <FooterBox>
+        <ConatinerFooter>
+          <Grid2 container columnSpacing={5}>
+            <Grid2 size={3}>
+              <Box>
+                <Typography variant="h4" sx={{ color: "white", mb: 2 }}>
```
Comment (tw-vineet) [inline client/src/Layout/Footer.tsx:46]: why using inline styling use to style components
 all inline style remove

File: client/src/index.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+import "./index.css"
+
+import reportWebVitals from "./reportWebVitals"
+import App from "./App"
+import { BrowserRouter } from "react-router-dom"
+import { ThemeProvider, createTheme } from "@mui/material"
+
+const root = ReactDOM.createRoot(document.getElementById("root") as HTMLElement)
+
+// const theme = createTheme({
```
Comment (tw-vineet) [inline client/src/index.tsx:12]: remove comment code

File: client/src/Layout/Footer.tsx [hunk 6]
```diff
@@ -0,0 +1,178 @@
+import { Box, Container, Grid2, Typography ,Divider} from "@mui/material"
```
Comment (shubh123thoughtwin) [inline client/src/Layout/Footer.tsx:1]: Imports of react should be at top, then other library import and at bottom component imports should place.Please check below example.

import React from "react"
import FacebookIcon from "@mui/icons-material/Facebook"
import { styled } from '@mui/system'

import Input from "../component/Input"
import Qrimagefooter from "../image/Frame 719.png"

File: client/src/component/Input.tsx [hunk 7]
```diff
@@ -0,0 +1,41 @@
+import { TextField } from "@mui/material"
+
+
+
+interface props {
```
Comment (shubh123thoughtwin) [inline client/src/component/Input.tsx:5]: First latter of interface should be capital.

File: client/src/Layout/Footer.tsx [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+                  ></Input>
+
+                  <SendIcon sx={{ color: "white", mr: 2 }} />
+                </FooterInput>
+              </Box>
+            </Grid2>
+
+            <Grid2 size={2}>
+              <Box>
+                <Typography variant="h5" sx={{ color: "white", mb: 2 }}>
```
Comment (shubh123thoughtwin) [inline client/src/Layout/Footer.tsx:82]: If Typography's css property are same you can make styled componet for that.

File: client/src/Layout/Footer.tsx [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+                  Support
+                </Typography>
+                <Typography variant="subtitle2" sx={{ color: "white", mb: 2 }}>
```
Comment (shubh123thoughtwin) [inline client/src/Layout/Footer.tsx:85]: Try to avoid repeat the code.

File: client/src/App.tsx (hunk 1 above)
Comment (adi061001) [reply client/src/App.tsx:16]: comment code is removed

File: client/src/App.css
Comment (adi061001) [reply client/src/App.css:1]: done

File: client/src/Layout/Footer.tsx (hunk 3 above)
Comment (adi061001) [reply client/src/Layout/Footer.tsx:22]: all the styled component is change into theme arrow function

File: client/src/index.tsx (hunk 5 above)
Comment (adi061001) [reply client/src/index.tsx:12]: done

File: client/public/logo512.png
Comment (adi061001) [reply client/public/logo512.png:1]: done

File: client/src/Layout/Footer.tsx (hunk 6 above)
Comment (adi061001) [reply client/src/Layout/Footer.tsx:1]: All the imports are now proper as you said

File: client/src/Layout/Footer.tsx (hunk 8 above)
Comment (adi061001) [reply client/src/Layout/Footer.tsx:82]: Done

File: client/src/Layout/Footer.tsx (hunk 9 above)
Comment (adi061001) [reply client/src/Layout/Footer.tsx:85]: ohk

File: client/src/component/Input.tsx (hunk 7 above)
Comment (adi061001) [reply client/src/component/Input.tsx:5]: ohk

File: client/src/Layout/Footer.tsx (hunk 4 above)
Comment (adi061001) [reply client/src/Layout/Footer.tsx:46]: all the inline css is removed and changed  it into style component

**Pull request closed without merging**